release-notes/major9/09.25.0/valtimo-frontend-libraries.md

```diff
@@ -34,6 +34,17 @@ The following features were added:
 
   For more information on this feature, see [this page](/using-valtimo/document/configuring-search-fields.md).
 
+* **Add `boolean` and `datetime` type for search fields configuration**
+
+  Added two new types for configuring search fields.
+  
+  The `boolean` field provides a selector with the following options: `yes`, `no` and `either`.
```

Comment:
The either option is removed. Talked with Thomas; with a dropdown leaving it empty equals to either. This situation was thought out for a radio button.